lilypad/lilypad-testnet/install-run-requirements.md

```diff
@@ -94,3 +94,11 @@ Use "lilypad [command] --help" for more information about a command.
 ```
 
 Thats it! You've successfully installed the Lilypad CLI on your machine! [🎉](https://emojipedia.org/party-popper)
+
+## Uninstall Lilypad
+
+To uninstall Lilypad, run the following command:
+
```

Comment:
I would add a callout (hint) similar to the one from the Linux RP docs:

`Please note that using sudo rm -rf is very powerful and can be dangerous if not used carefully.`